flux/stdlib/influxdata/influxdb/buckets.go

```diff
@@ -126,3 +141,27 @@ type MetaClient interface {
 	Databases() []meta.DatabaseInfo
 	Database(name string) *meta.DatabaseInfo
 }
+
+type LocalBucketsRule struct{}
+
+func (rule LocalBucketsRule) Name() string {
+	return "influxdata/influxdb.LocalBucketsRule"
+}
+
+func (rule LocalBucketsRule) Pattern() plan.Pattern {
+	return plan.Pat(influxdb.BucketsKind)
+}
+
+func (rule LocalBucketsRule) Rewrite(ctx context.Context, node plan.Node) (plan.Node, bool, error) {
```

Comment:
Consider naming the return values for clarity.

Reply:
Copied from 2.x so will keep for now